### PR TITLE
Add footer-link to ContactInfo

### PR DESCRIPTION
**Pull request by Sosnovskyi-Roman**
I added footed-link 
Screenshot 2024-07-06 at 21 59 49

### CODE REVIEW

File: src/components/MainPage/Footer/Sections/ContactInfo.jsx [hunk 1]
```diff
@@ -1,24 +1,43 @@
...
-          <a href=''><img src={MessengerIcon} className='social-icon'/></a> 
-          <a href=''><img src={TelegramIcon} className='social-icon'/></a>
-          <a href=''><img src={FacebookIcon} className='social-icon'/></a> 
-          <a href=''><img src={EmailIcon} className='social-icon'/></a> 
-          <a href=''><img src={PhoneIcon} className='social-icon'/></a> 
-        </nav>
+    <div className="contact-section">
+      <h3>Contact Information</h3>
+      <nav className="contact-nav">
+        <a href="">
```
Comment (mskmee) [inline src/components/MainPage/Footer/Sections/ContactInfo.jsx:15]: Lets use constant here.
It's supposte to be an array of object with structure:
`
[
{
icon: string,
className: string, 
href: string,
target: string
}
]
`
We should map this object here instead

**Review by mskmee — CHANGES_REQUESTED** [review]

**Comment by mskmee** [comment]
Merge dev branch please, and use prettier config, that we have in the project

File: src/components/MainPage/Footer/Sections/ContactInfo.jsx (hunk 1 above)
Comment (Sosnovskyi-Roman) [reply src/components/MainPage/Footer/Sections/ContactInfo.jsx:15]: Something like this?
import React from "react";
import MessengerIcon from "../social-media/messenger.png";
import InstagramIcon from "../social-media/insta.png";
import TelegramIcon from "../social-media/telegram.png";
import FacebookIcon from "../social-media/facebook.png";
import EmailIcon from "../social-media/email.svg";
import PhoneIcon from "../social-media/phone.png";

const contactInfo = [
  {
    icon: InstagramIcon,
    className: "social-icon",
    href: "https://www.instagram.com",
    target: "_blank",
  },
  {
    icon: MessengerIcon,
    className: "social-icon",
    href: "https://www.messenger.com",
    target: "_blank",
  },
  {
    icon: TelegramIcon,
    className: "social-icon",
    href: "https://telegram.org",
    target: "_blank",
  },
  {
    icon: FacebookIcon,
    className: "social-icon",
    href: "https://www.facebook.com",
    target: "_blank",
  },
  {
    icon: EmailIcon,
    className: "social-icon",
    href: "mailto:someone@example.com",
    target: "_blank",
  },
  {
    icon: PhoneIcon,
    className: "social-icon",
    href: "tel:+123456789",
    target: "_blank",
  },
];

export default function ContactInfo() {
  return (
    <div className="contact-section">
      <h3>Contact Information</h3>
      <nav className="contact-nav">
        {contactInfo.map((item, index) => (
          <a
            key={index}
            href={item.href}
            target={item.target}
            rel="noopener noreferrer"
          >
            
          </a>
        ))}
        <a
          href="https://teamchallenge.io/team/387/public"
          className="footer-link"
          target="_blank"
          rel="noopener noreferrer"
        >
          © 2024 GadgetHouse
        </a>
      </nav>
    </div>
  );
}

Comment (Sosnovskyi-Roman) [reply src/components/MainPage/Footer/Sections/ContactInfo.jsx:15]: Screenshot 2024-07-07 at 18 36 12
Screenshot 2024-07-07 at 18 36 48
Something like this?

**Pull request merged**